GoogleChromeLabs/ps-analysis-tool
# Feature: Use `cookieStore` API to fetch JS cookies.

## Description
This PR aims to change the method used by PSAT to fetch the `JavascriptCookie`.

## Relevant Technical Choices



## Testing Instructions
- Clone this PR and in the terminal run `npm start`.
- Open a fresh Chrome profile and install the PSAT extension.
- Add a javascript cookie from the command line and now you should see the cookie on the PSAT DevTools page.

## Additional Information:
- The `document.cookie` has been replaced with `cookieStore` API.

## Screenshot/Screencast



---

## Checklist
